AlphaWallet/TokenScriptClient/Models/XMLHandler.swift

```diff
@@ -367,7 +367,7 @@ private class PrivateXMLHandler {
             }
         case .unsupportedTokenScriptVersion(let isOld):
             if isOld {
-                assertImpossibleCodePath()
+                //assertImpossibleCodePath()
```

Comment:
Does the code reach here at runtime? Sounds like a bug.

Reply:
Yes, if it is old or too new it will hit the assert and crash the app